src/components/20-molecules/footer-small/index.js

```diff
@@ -100,16 +91,24 @@ class AXAFooterSmall extends InlineStyles {
             <ul class="m-footer-small__list">
               ${repeat(
                 this.querySelectorAll('[slot="language-item"]'),
-                (languageItem, index) => html`
-                  <li class="m-footer-small__list-item">
-                    <a
-                      href="${languageItem.href}"
-                      class="m-footer-small__link--bold"
-                      @click=${ev => this.handleLanguageClick(ev, index)}
-                      >${languageItem.textContent}</a
-                    >
-                  </li>
-                `
+                (languageItem, index) => {
+                  const classes = {
+                    'm-footer-small__link--bold': true,
+                    'm-footer-small__link--active': languageItem.classList.contains(
```

Comment:
This line initially  leaves one puzzled, until we understand you are copying a lightDOM element to inside the ShadowDOM, and want to copy relevant classes, too. Comment might be helpful...

Reply:
I will add the comment to the release branch. Because I will lose the approval when I put another commit.